src/filters/truncate.js

```diff
@@ -21,8 +21,12 @@ angular.module('openshiftCommonUI')
       if (useWordBoundary !== false) {
         // Find the last word break, but don't look more than 10 characters back.
         // Make sure we show at least the first 5 characters.
-        var startIndex = Math.max(4, charLimit - 10);
-        var lastSpace = truncated.lastIndexOf(/\s/, startIndex);
+        if (truncated.length > charLimit) {
```

Comment:
Indeed it doesn't fire in most cases, correct... looks like it needs to be corrected because the only way it returns a value is when charLimit is ```null``` which is kind of useless, but the intent appears to be related to how the "hard truncate" is separate from "new line", and "word boundary". Regardless this does help highlight the use of the "-10" characters.

On an earlier version of this we briefly had the "-10" as a parameter with a default of "-10" we could go back to that, or revert it entirely to maintain part of the original behavior?